cli/tests/integrations/test_task.py

```diff
@@ -328,3 +328,15 @@ def _get_completed_task_id(app_id='test-app-completed'):
     assert len(tasks) == 1
     task_id = tasks[0]['id']
     return task_id
+
+
+def _read_lines_once_available(file_obj):
```

Comment:
rename to `_read_lines(file_obj)`, we don't need to expose implementation. Please add a pydoc :)